ports/sleepy-discord/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+  FEATURES
+    "websocketpp"    USE_WEBSOCKETPP
+    "websocketpp"    USE_BOOST_ASIO
+    "cpr"            USE_CPR
+    "voice"          ENABLE_VOICE
+    "compression"    USE_ZLIB
```

```suggestion
vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
    FEATURES
        "websocketpp"    USE_WEBSOCKETPP
        "websocketpp"    USE_BOOST_ASIO
        "cpr"            USE_CPR
        "voice"          ENABLE_VOICE
        "compression"    USE_ZLIB
```
It might be better to place this part below `vcpkg_from_github()`.